src/help/help.es6

```diff
@@ -0,0 +1,305 @@
+			}, {
+				"text": "Commodities",
+				"url": "commodities.html"
+			}, {
+				"text": "Contract period",
+				"url": "contract-period.html"
+			}, {
+				"text": "Derivative",
+				"url": "derivative.html"
+			}, {
+				"text": "Duration",
+				"url": "duration.html"
+			}, {
+				"text": "Ends Between/Ends Outside trades",
+				"url": "ends-between.html"
+			}, {
+				"text": "Entry spot price",
+				"url": "entry-spot.html"
+			}, {
+				"text": "Expiry price",
+				"url": "expiry-price.html"
+			}, {
+				"text": "Forex",
+				"url": "forex.html"
+			}, {
+				"text": "GMT",
+				"url": "gmt.html"
+			}, {
+				"text": "Higher/Lower trades",
+				"url": "h_l-trades.html"
+			}, {
+				"text": "Indices",
+				"url": "indices.html"
+			}, {
+				"text": "In/Out trades",
+				"url": "i_o-trades.html"
+			}, {
+				"text": "Market exit price",
+				"url": "m_exit-price.html"
+			}, {
+				"text": "No Touch trades",
+				"url": "no-touch-trades.html"
+			}, {
+				"text": "(One) Touch trades",
+				"url": "touch-trades.html"
+			}, {
+				"text": "Payout",
+				"url": "payout.html"
+			}, {
+				"text": "Pip",
+				"url": "pip.html"
+			}, {
+				"text": "Profit",
+				"url": "profit.html"
+			}, {
+				"text": "Volatility Indices",
+				"url": "volatility-indices.html"
+			}, {
+				"text": "Resale price",
+				"url": "resale-price.html"
+			}, {
+				"text": "Return",
+				"url": "return.html"
+			}, {
+				"text": "Rise/Fall trades",
+				"url": "r_f-trades.html"
+			}, {
+				"text": "Sell option",
+				"url": "sell-option.html"
+			}, {
+				"text": "Spot price",
+				"url": "spot-price.html"
+			}, {
+				"text": "Stake",
+				"url": "stake.html"
+			}, {
+				"text": "Stays Between/Goes Outside trades",
+				"url": "stays-between-goes-outside-trades.html"
+			}, {
+				"text": "Tick",
+				"url": "tick.html"
+			}, {
+				"text": "Underlying",
+				"url": "underlying.html"
+			}]
+		}
+	};
+
+	state.updateSublist = (list) => {
+		state.current.list = list;
+		state.current.sublist = state.sublist[list.sublist_id];
+		state.getContent(state.current.sublist[0].url);
+	};
+
+	state.getContent = (url) => {
+		state.current.loading = true;
+		state.current.content_page = url;
+		require(['text!help/'+url],(content) => {
+			state.current.loading = false;
+			state.current.content = content;
+			//Modify links in content to open in new tab.
+			$(".help-dialog .content").find("a").each((i, node)=>{
+				$(node).attr('target','_blank');
+			});
+		});
+	};
+
+	state.searchSublist = (e,scope) => {
+		var query = $(e.target).val().toLowerCase();
+		if(query.length > 0){
+			state.current.sublist = sublist_items.filter((item) => {
+				return item.text.toLowerCase().indexOf(query) != -1;
+			});
+			state.current.sublist && state.current.sublist.length && state.getContent(state.current.sublist[0].url);
+		}
+	}
+
+	//Concat all the sublist items into one array so that we can later use it for searching.
+	for (var key in state.sublist){
```

Comment:
@4p00rv In ES6, please do not use `var`. `var` has got some other side effects. Use `const` or `let` as per situation